dask_expr/_collection.py

```diff
@@ -346,8 +346,8 @@ def index(self, value):
     def reset_index(self, drop=False):
         return new_collection(expr.ResetIndex(self, drop))
 
-    def head(self, n=5, compute=True):
-        out = new_collection(expr.Head(self, n=n))
+    def head(self, n=5, compute=True, npartitions=1):
```

Comment:
Can you switch npartitions and compute to avoid changing the argument order?

Reply:
Oh ya, good catch. https://github.com/dask-contrib/dask-expr/pull/636/commits/536fc36e22578ee79f37004452fbb7a7ffdd63f0